r/rsparkling/R/h2o_context.R

```diff
@@ -1,14 +1,14 @@
 #' Get the H2OContext. Will create the context if it has not been previously created.
 #'
 #' @param x Object of type \code{spark_connection} or \code{spark_jobj}.
-#'
+#' @param strict_version_check (Optional) Setting this to FALSE does not cross check version of H2O and attempts to connect.
 #' @export
-h2o_context <- function(x) {
+h2o_context <- function(x, strict_version_check = TRUE) {
   UseMethod("h2o_context")
 }
 
 #' @export
-h2o_context.spark_connection <- function(x) {
+h2o_context.spark_connection <- function(x, strict_version_check = TRUE) {
```

Comment:
argument `strict_version_check` was not passed in a function few lines below this one.